core/trino-spi/src/main/java/io/trino/spi/connector/RowChangeParadigm.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.connector;
+
+public enum RowChangeParadigm
+{
+    ROW_CHANGE_NOT_SUPPORTED,
+    // The JDBC paradigm - - requires just the rowId and the changed columns
```

Comment:
convert into javadoc and start with the definition. Then you can refer to JDBC as an example

Reply:
Changed.